charts/pulsar/templates/proxy/proxy-configmap.yaml

```diff
@@ -59,10 +59,11 @@ data:
   # Authentication Settings
   {{- if .Values.auth.authentication.enabled }}
   authenticationEnabled: "true"
+  # this configuration should be always true when authentication is enabled
+  forwardAuthorizationCredentials: "true"
```

Comment:
Whether you have enabled `authenticateOriginalAuthData` in the broker configuration.

Reply:
yes, the chart enabled the `authenticateOriginalAuthData` when authentication is enabled: https://github.com/streamnative/charts/blob/master/charts/pulsar/templates/broker/broker-configmap.yaml#L118